lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +22,13 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    tokens = []
+    for symbol in text:
+        if symbol.isalpha():
```

Comment:
if not - continue

Reply:
done

Reply:
Actually not. revert the condition to `if not symbol.isalpha():` -> so, you can moce `continue` to the if statement.  And `else` word can be deleted